.github/workflows/pull_request.yml

```diff
@@ -8,6 +8,9 @@ on:
       - 'enhancements/**'
       - '*.md'
 
+permissions:
```

Comment:
https://github.com/sustainable-computing-io/kepler/actions/runs/6224518606
I suppose scord card... may has a long way to go, does anyone know where to report/open ticket for them?